pkg/translator/jaeger/jaegerproto_to_traces_test.go

```diff
@@ -416,6 +416,15 @@ func TestSetInternalSpanStatus(t *testing.T) {
 			status:           errorStatus,
 			attrsModifiedLen: 1,
 		},
+		{
+			name: "do not set any error tag if one is already present",
+			attrs: map[string]interface{}{
+				tracetranslator.TagError:            true,
+				conventions.AttributeHTTPStatusCode: 404,
+			},
+			status:           errorStatus,
```

Comment:
I must have read the code wrongly, as I would have expected this test to have failed. Given this test passes, the behavior is validated. The only other case that needs to be tested (in another PR perhaps?) is to set the error tag if the status code is greater than 500 and no error tag is set yet.